### DISCUSSION TITLE
GET 조회 성공시에 data가 없는 경우. Data 처리

### DISCUSSION THREAD

**Discussion by unanchoi** [Katchup-dev/Katchup-server#31] [General]
https://github.com/Katchup-dev/Katchup-server/pull/27#pullrequestreview-1431830327

이 부분 관련해서 논의가 필요한 것 같습니다!
제 기억에는 GET을 제외한 다른 API에서는 받아올 data가 필요 없는 상황에서는
```JSON
{
  status:  
  success:
  message:
}
```
아래 3개만 표시하기로 되어있는데, 현재 상황은 GET에서 조회는 성공했으나, 조회할 data가 없는 상황에서 어떻게 처리하는 상황인 것 같습니다. 이 부분에 대해서 제 기억에는 data가 없으면 별도로 204 NO CONTENT 없이 200  OK로 응답하되

```JSON
{
  status:  
  success:
  message:
  data: null

}
```
아래 같이 작성하기로 했던 것 같은데, 혹시 맞는지 check 좀 부탁드립니다.

**Comment by Seokyeong237** [comment]
저도 이렇게 기억합니다! data: null 이 있어야 조회는 성공했는데 data가 없는 거구나 라고 인식이 될 수 있을 것 같아요!

**Comment by yeseul106** [comment]
음 사실 GET의 상황에서 조회할 data가 없을 때를 제외한 나머지 케이스를 중점으로 생각해서 3가지만 내려줄 수 있도록 해당 어노테이션을 사용했던 것이긴 합니다. 

근데 조회할 data가 없는 케이스도 생각해보면, success 값과 message로 성공 여부를 알려주니 data 필드가 없더라도 실패가 아닌 조회할 데이터가 없다고 판단할 수 있을 것이라고 생각이 들긴 하는데.. 직관적인 것을 생각했을 때는 두분 말대로 data:null이 있는 것이 더 클라분들이 이해하기 쉬울 수도 있겠네요 !!
